src/test/java/com/jcabi/log/ParseLevelInformationTest.java

```diff
@@ -0,0 +1,106 @@
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.log;
+
+import java.util.concurrent.ConcurrentHashMap;
+import org.apache.commons.lang3.StringUtils;
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Test case for {@link ParseLevelInformation}.
+ * @author Jose V. Dal Pra Junior (jrdalpra@gmail.com)
+ * @version $Id$
+ * @checkstyle MultipleStringLiteralsCheck (80 lines)
+ */
+public class ParseLevelInformationTest {
+
+    /**
+     * Test the default path.
+     */
+    @Test
+    public final void mustParseCorrectInformation() {
+        final ConcurrentHashMap<String, String> parsed =
+            new ParseLevelInformation("INFO:2;10,WARN:2;32").parse();
+        Assert.assertThat(parsed, Matchers.hasEntry("INFO", "2;10"));
+        Assert.assertThat(parsed, Matchers.hasEntry("WARN", "2;32"));
+    }
+
+    /**
+     * Test the wrong pattern path.
+     */
+    @Test
+    public final void mustNotParseIncorrectInformation() {
+        try {
+            final ConcurrentHashMap<String, String> parsed =
+                new ParseLevelInformation("INFO;10,WARN;32").parse();
+            Assert.assertThat(
+                "Never should enter this assert!",
+                parsed,
+                Matchers.nullValue()
+            );
+        } catch (final IllegalStateException ex) {
+            Assert.assertThat(
+                ex.getMessage(), Matchers.equalToIgnoringCase(
+                    String.format(
+                        StringUtils.join(
+                            "Information is not using the pattern ",
+                            "KEY1:VALUE,KEY2:VALUE %1s"
+                            ),
+                        "INFO;10,WARN;32"
+                    )
+                )
+            );
+        }
+    }
+
+    /**
+     * Test the wrong {@link org.apache.log4j.Level} type path.
+     */
+    @Test
+    public final void mustNotParseInformationWithWrongLevelType() {
+        try {
+            final ConcurrentHashMap<String, String> parsed =
+                    new ParseLevelInformation("INFO:2;10,EXTREME:2;32").parse();
+            Assert.assertThat(
+                "Never should enter this assert!",
+                parsed,
+                Matchers.nullValue()
```

Comment:
@jrdalpra this is less than 80 chars, please read http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#indentation
```
                parsed, Matchers.nullValue()
```